tests/testthat/test-utils.R

```diff
@@ -5,3 +5,49 @@ testthat::test_that("get_bs_version", {
 testthat::test_that("panel_item", {
   testthat::expect_s3_class(panel_item("LABEL", shiny::tags$div()), "shiny.tag")
 })
+
+testthat::test_that("Test to_flextable: supported class", {
+  data_frame <- data.frame(A = 1:3, B = 4:6)
+  flextable_output <- to_flextable(data_frame)
+  testthat::expect_s3_class(flextable_output, "flextable")
+})
+
+testthat::test_that("Test to_flextable: unsupported class", {
+  unsupported_data <- list(a = 1, b = 2)
+  expect_error(to_flextable(unsupported_data), "Unsupported class")
+})
+
+test_that("custom theme to flextable", {
```

Comment:
To keep test naming consistent (start with function name)

```suggestion
test_that("custom_theme to flextable", {
```